src/js/components/state/spendingovertime/StateTimeVisualization.jsx

```diff
@@ -67,7 +68,7 @@ export default class StateTimeVisualization extends React.Component {
 
         const legend = [
             {
-                color: '#708893',
+                color: this.props.color,
```

Comment:
I think having this variable breaks the uniformity of the code here, since all the other components explicitly declare the color hexcode. Could we change this back to the hex code or have all the others use the variable?